grails-forge-core/src/main/java/org/grails/forge/feature/test/Geb.java

```diff
@@ -105,24 +105,29 @@ public void apply(GeneratorContext generatorContext) {
                 .artifactId("geb")
                 .test());
 
+        generatorContext.addDependency(Dependency.builder()
+                .groupId("org.gebish")
+                .artifactId("geb-spock")
+                .test());
+
         Stream.of("api", "support", "remote-driver")
                 .map(name -> "selenium-" + name)
                 .forEach(name -> generatorContext.addDependency(Dependency.builder()
                         .groupId("org.seleniumhq.selenium")
-                        .lookupArtifactId(name)
+                        .artifactId(name)
                         .test()));
 
         generatorContext.addDependency(Dependency.builder()
                 .groupId("org.seleniumhq.selenium")
-                .lookupArtifactId("selenium-chrome-driver")
+                .artifactId("selenium-chrome-driver")
```

Comment:
`selenium-chrome-driver` is already a transitive dependency from `testFixtures('org.grails.plugins:geb')`. However, `selenium-firefox-driver` and `selenium-safari-driver` are currently not.

Reply:
removed.